### PR TITLE
added sparkinfo command and logging

### PR DESCRIPTION
**Pull request by jbloom22**
- number of partitions and storage level are logged for each command that requires vds

### CODE REVIEW

File: src/main/scala/org/broadinstitute/hail/driver/Command.scala [hunk 1]
```diff
@@ -249,8 +250,11 @@ abstract class Command {
       fatal("this module requires a VDS.\n  Provide a VDS through a `read' or `import' command first.")
     else if (!supportsMultiallelic && !state.vds.wasSplit)
       fatal("this module does not support multiallelic variants.\n  Please run `splitmulti' first.")
-    else
+    else {
+      if (requiresVDS)
+        log.info(s"sparkinfo: ${state.vds.nPartitions} partitions, ${state.vds.rdd.getStorageLevel}")
```
Comment (tpoterba) [inline src/main/scala/org/broadinstitute/hail/driver/Command.scala]: add $name after sparkinfo -- will be a little clearer.

**Comment by jbloom22** [comment]
I added the command name, and switched to printing the readable string rather than the `StorageLevel(true, false, false, false, 1)` by adding a RichStorageLevel with toReadableString().  Also added a test in UtilsSuite

**Comment by jbloom22** [comment]
Rebased, back to you.

**Comment by tpoterba** [comment]
Will merge when tests pass.  Make a post on discourse updates to introduce the new command!

**Pull request merged**